rust-runtime/aws-smithy-http-server/src/lib.rs

```diff
@@ -30,6 +30,9 @@ pub use self::routing::Router;
 #[doc(inline)]
 pub use tower_http::add_extension::{AddExtension, AddExtensionLayer};
 
+#[doc(inline)]
+pub use aws_smithy_http::byte_stream::ByteStream;
```

Comment:
@david-perez @crisidev Should I keep this re-export of ByteStream in the server?

Reply:
I think this is fine for now. We are discussing about creating our own BytesStream object vs using the SDK provided one. 

Having the export IMO moves us in the right direction even if we will change the implementation in the near future.

Thoughts @david-perez?

I think it should not be the aim of this PR to sort out the ByteStream implementation but rather have an initial implementation of the codegen needed for streaming types.

Reply:
@crisidev 

> Thoughts @david-perez?

https://github.com/awslabs/smithy-rs/pull/1023#discussion_r784751006

> I think it should not be the aim of this PR to sort out the ByteStream implementation but rather have an initial implementation of the codegen needed for streaming types.

OK. Let's track that here https://github.com/awslabs/smithy-rs/issues/1125.